ou_dedetai/cli.py

```diff
@@ -68,6 +70,10 @@ def run_indexing(self):
 
     def run_installed_app(self):
         self.logos.start()
+        # Keep the process running so that our background threads can keep running
+        while self.logos.logos_state != LogosRunningState.STOPPED:
+            time.sleep(10)
+            self.logos.monitor()
```

Comment:
Can this be achieved instead by using `daemon=False` on the long-running threads?

Reply:
Don't think so, if the CLI process exits the the threads spawned by python terminate